pkg/apis/experimental/doc.go

```diff
@@ -17,7 +17,7 @@ limitations under the License.
 // +groupName=experimental.cert-manager.io
 // +groupGoName=Experimental
 
-// Package certmanager is the internal version of the API.
+// Package experimental is the internal version of the API.
```

Comment:
nitpick(non-blocking): the package name is correct now, but the comment itself doesn't refer to what this package does; it's still copy-pasted.